packages/dev-frontend/src/ThresholdFrontend.tsx

```diff
@@ -25,6 +25,45 @@ type ThresholdFrontendProps = {
   loader?: React.ReactNode;
 };
 
+async function connectToSepolia() {
+  if (window.ethereum) {
+    try {
+      const ethereum = window.ethereum;
+      const accounts = await ethereum.request({
+        method: "eth_requestAccounts"
+      });
+
+      // Log public address of user
+      console.log(accounts[0]);
+
+      // Get network ID
+      let n = ethereum.chainId;
+
+      // Request adding the custom network
+      let params = [
+        {
+          chainId: "0xAA36A7",
+          chainName: "Sepolia test network",
+          rpcUrls: ["https://rpc.sepolia.org"],
```

Comment:
If this is the RPC URL we're using for Sepolia, do we need to include the Infura API key in the frontend anymore?

Reply:
Discussed offline